template/src/ontology/Makefile.jinja2

```diff
@@ -1130,7 +1131,7 @@ $(ONT)-non-classified.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC) $(IMPORT_FILES)
 $(ONT)-simple.owl: $(EDIT_PREPROCESSED) $(OTHER_SRC) $(SIMPLESEED) $(IMPORT_FILES)
 	$(ROBOT_RELEASE_IMPORT_MODE) \{% if project.release_use_reasoner %}
 		reason --reasoner {{ project.reasoner }} --equivalent-classes-allowed {{ project.allow_equivalents }} --exclude-tautologies {{ project.exclude_tautologies }} --annotate-inferred-axioms {{ project.release_annotate_inferred_axioms|default('false')|lower }} \{% endif %}
-		relax \
+		relax $(RELAX_OPTIONS) \
 		remove --axioms equivalent \
 		relax \
```

Comment:
Unrelated to this issue: Not sure why this second `relax` command is there? What could it relax that has not already been relaxed by the first command?

Reply:
I made this comment in a separate context; I cant imagine it doing anything at all! `reduce` makes sense, a second relax does not.